backfill_corrections/delphiBackfillCorrection/R/beta_prior_estimation.R

```diff
@@ -146,13 +146,18 @@ frac_adj <- function(train_data, test_data, prior_test_data,
                      train_models = TRUE,
                      make_predictions = TRUE) {
   train_data$value_target <- frac_adj_with_pseudo(train_data, NULL, 1, 100, "value_target_num", "value_target_denom")
-  train_data$value_7dav <- frac_adj_with_pseudo(train_data, NULL, 1, 100, "value_7dav_num", "value_7dav_denom")
-  prior_test_data$value_7dav <- frac_adj_with_pseudo(prior_test_data, NULL, 1, 100, "value_7dav_num", "value_7dav_denom")
-  
   train_data$log_value_target <- log(train_data$value_target)
+
+  test_data$value_target <- frac_adj_with_pseudo(test_data, NULL, 1, 100, "value_target_num", "value_target_denom")
+  test_data$log_value_target <- log(test_data$value_target)
```

Comment:
@jingjtang I added these definitions to get the pipeline to run. Fraction test data didn't have any `value_target_*` fields defined so it would fail [during evaluation](https://github.com/cmu-delphi/covidcast-indicators/blob/efc5f5886d7134c46b743033df5fcf5d16a4f380/backfill_corrections/delphiBackfillCorrection/R/model.R#L165). Wanted to run this change by you.

Because of this change, I had to add some fields to the fake test data in [test-beta_prior_estimation.R](https://github.com/cmu-delphi/covidcast-indicators/pull/1704/files#diff-d6f041733ee06a2788aa41f47dca9e6dd431d5cd5fd825cfa02f00a815f85c6d).

Reply:
It's definitely a bug. Thanks for pointing it out.